CodeCoverageQualityGate/CodeCoverageQualityService.ts

```diff
@@ -112,10 +113,10 @@ export class CodeCoverageQualityService {
             if (!this.readReferenceBuildInfo()) {
                 return;
             }
-            // Get Cobertura report in current build
-            const currentCoberturaReport = await this.getCoberturaReportOfCurrentBuild();
+            // Get Merged Cobertura report in current build
+            const currentCoberturaReport = await this.getMergedCoberturaReportOfCurrentBuild();
             if (!currentCoberturaReport) {
-                this.skipQualityGateWithWarning(`no Parasoft coverage results were found in this build`);
+                this.skipQualityGateWithWarning(`no merged Parasoft coverage report is found in this build`);
```

Comment:
I don't think we should modify the message. If we get nothing at this point, it means we haven't found any coverage results. If there's a conflict during the merge, at least the first report will be retained.

Reply:
Please reconsider other places with similar messages as well.